picard/formats/id3.py

```diff
@@ -659,6 +659,11 @@ def _remove_deleted_tags(self, metadata, tags):
             except KeyError:
                 pass
 
+        if self.metadata.images._deleted:
+            for key, frame in list(tags.items()):
```

Comment:
Is `list()` needed here?

Reply:
It doesn't seem to be needed. I will remove it.